lib/routes/dev/dev.dart

```diff
@@ -418,8 +419,15 @@ class DevViewState extends State<DevView> {
           encoder.close();
           ShareExtend.share(zipFile, "file");
         }));
+    
         */
-
+    choices.add(Choice(
+        title: 'Reset POS DB',
+        icon: Icons.phone_android,
+        function: () {
+          PosCatalogBloc bloc = AppBlocsProvider.of<PosCatalogBloc>(context);
+          bloc.resetDB();
+        }));
     return choices;
```

Comment:
I know most items on 'Developers' screen is not for users or not ready for their use yet but I think this option does not belong here. Since 'Developers' is not accesible from POS mode. I believe "Reset POS DB" should be moved to POS Settings. It should have several warning/confirmation dialogs. Enter Manager Password, PIN to confirm etc. 
At the same time, I believe we should wait till we can import items from any source first before making this visible to users..

Reply:
I only did this to be used internally because I changed the db scheme. It is not meant for uses and I will remove it once the POS is ready.